packages/neutrino-middleware-eslint/package.json

```diff
@@ -20,14 +20,10 @@
     "eslint": "^4.7.2",
     "eslint-loader": "^1.9.0",
     "eslint-plugin-babel": "^4.1.2",
-    "eslint-plugin-import": "^2.7.0",
     "fluture": "^7.1.3",
     "lodash.clonedeep": "^4.5.0",
     "ramda": "^0.24.1"
   },
-  "devDependencies": {
-    "eslint-config-airbnb-base": "^12.0.1"
```

Comment:
In the past the reason this devDependency existed had to do with local development and linking. ESLint didn't like to resolve where this package existed for the Airbnb preset when you linked it, probably due to the nested node_modules structure. My guess is that problem still exists, but can probably be fixed if we move to yarn workspaces and hoisted deps.

Reply:
I agree there is a bug in yarn where it's over-eager to complain about "missing" peer deps (I'm going to file that separately). However in this case, the `devDependency` was actually in the wrong package - it's airbnb base that's using it :-)

Reply:
Yeah, that's what I mean. When you link `neutrino-preset-airbnb-base`, ESLint couldn't resolve `eslint-config-airbnb-base` because the eslint binary lives in `neutrino-middleware-eslint` but it was trying to use a working directory relative to `neutrino-preset-airbnb-base`.

Reply:
Ah yeah that makes sense. I thought you meant a related bug that I ran into, where yarn outputs "missing peer" for a case that npm doesn't.

I'm looking forward to yarn workspaces working well and all of this going away! :-)

Reply:
Me too Ed, me too.